### ISSUE TITLE
Instagram stories

### ISSUE THREAD

**Issue by turkeyphant** [yt-dlp/yt-dlp#2332]
### Checklist

- [X] I'm reporting a broken site
- [X] I've verified that I'm running yt-dlp version **2021.12.27**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

UK

### Description

ERROR: Unsupported URL: https://www.instagram.com/stories/[xxx]

Works fine for Instagram feed posts, just not stories.
Cookies does not help.

### Verbose log

```shell
yt-dlp https://www.instagram.com/stories/[xxx] --cookies instacookie.txt -Uv
[debug] Command-line config: ['-i', '--format', '((571/272/402/337/315/313/401/336/308/400/271/335/303/299/399/137/248/334/302/298/398/247/136/333/244/135/397/332/243/134/396/331/242/133/395/330/160/394/278)[protocol!=http_dash_segments])+(bestaudio[acodec=opus]/bestaudio[protocol!=http_dash_segments])/best', 'https://www.instagram.com/stories/[xxx]', '--cookies', 'instacookie.txt', '-Uv']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.12.27 [6223f67a8] (zip)
[debug] Python version 3.9.0 (CPython 64bit) - Linux-4.19.107-Unraid-x86_64-Intel-R-_Xeon-R-_CPU_E5-2650_v2_@_2.60GHz-with-glibc2.30
[debug] exe versions: ffmpeg N-54690-g682990a849-static
[debug] Optional libraries: sqlite
[debug] Proxy map: {}
Latest version: 2021.12.27, Current version: 2021.12.27
yt-dlp is up to date (2021.12.27)
[debug] [generic] Extracting URL: https://www.instagram.com/stories/[xxx]
[generic] [xxx]: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] [xxx]: Downloading webpage
[generic] [xxx]: Extracting information
[debug] Looking for video embeds
ERROR: Unsupported URL: https://www.instagram.com/stories/[xxx]
Traceback (most recent call last):
  File "/usr/local/bin/youtube-dl/yt_dlp/YoutubeDL.py", line 1353, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/bin/youtube-dl/yt_dlp/YoutubeDL.py", line 1423, in __extract_info
    ie_result = ie.extract(url)
  File "/usr/local/bin/youtube-dl/yt_dlp/extractor/common.py", line 599, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/youtube-dl/yt_dlp/extractor/generic.py", line 3829, in _real_extract
    raise UnsupportedError(url)
yt_dlp.utils.UnsupportedError: Unsupported URL: https://www.instagram.com/stories/[xxx]
```

**Comment by Ashish0804** [comment]
That page doesn't exist that's why it's failing.

**Comment by turkeyphant** [comment]
It obviously did exist at the time I opened this ticket. Stories only exist for 24 hours.

**Comment by pukkandan** [comment]
You are going to have to provide a URL

**Comment by nyuszika7h** [comment]
I guess you could use highlights like this one that don't disappear:

https://www.instagram.com/stories/highlights/17888807746396621/

**Comment by MinePlayersPE** [comment]
Already fixed on master https://github.com/yt-dlp/yt-dlp/commit/dd5e60b15dd720727a0242eec2bd6698cb03b8f2
```
>yt-dlp -F "https://www.instagram.com/stories/thxriqq/XXX/" --cookies-from-browser edge
[Cookies] Extracting cookies from edge
[Cookies] Extracted 2777 cookies from edge
[instagram:story] XXX: Downloading JSON metadata
[instagram:story] XXX: Downloading JSON metadata
[download] Downloading playlist: XXX
[instagram:story] playlist XXX: Collected 2 videos; downloading 2 of them
[download] Downloading video 1 of 2
[info] Available formats for XXX:
...
```

**Comment by nyuszika7h** [comment]
Yeah, just checked, it works fine for me too on master.

**Comment by turkeyphant** [comment]
> Already fixed on master [dd5e60b](https://github.com/yt-dlp/yt-dlp/commit/dd5e60b15dd720727a0242eec2bd6698cb03b8f2)
> 
> ```
> >yt-dlp -F "https://www.instagram.com/stories/XXX/XXX/" --cookies-from-browser edge
> [Cookies] Extracting cookies from edge
> [Cookies] Extracted 2777 cookies from edge
> [instagram:story] XXX: Downloading JSON metadata
> [instagram:story] XXX: Downloading JSON metadata
> [download] Downloading playlist: XXX
> [instagram:story] playlist XXX: Collected 2 videos; downloading 2 of them
> [download] Downloading video 1 of 2
> [info] Available formats for XXX:
> ...
> ```

Thanks! It has "Unsupported URL" with version 2021.12.27 so do you know when this is going to be pushed to an update?
